### PR TITLE
Removal of Motoman Packages

### PR DESCRIPTION
**Pull request by Jmeyer1292**
This PR removes a bunch of old and un-used packages from the current iteration of Godel, mostly related to our old test bench.

### CODE REVIEW

**Review by geoffreychiou — APPROVED** [review]
Fixed the merge conflict, builds fine, +1.

File: README.md [hunk 1]
```diff
@@ -49,15 +49,7 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   sim_robot:=false robot_ip:=[robot ip]
   ```
 
-- Bring up Kinect2 sensor, if available:
```
Comment (woodaaron) [inline README.md:52]: Probably a separate issue, but should we document supported hardware?

Also, we have a few references to the kinetic sensor that we've re-purposed for the ensenso (e.g. the static publisher from kinetic2_move_frame). If we are officially dropping support for the Kinect sensor, we should rename/remove those references.

**Review by woodaaron — COMMENTED** [review]
After code removal, project still builds and runs with IRB2400 and Ensenso. Does not seem to impact core functions.

File: README.md (hunk 1 above)
Comment (Jmeyer1292) [reply README.md:52]: I agree. This point kind of goes with issue #133.

**Pull request merged**